.golangci.yml

```diff
@@ -1,21 +1,25 @@
+version: "2"
 run:
-  deadline: 5m
+  timeout: 5m
   allow-parallel-runners: true
 
 linters:
   enable:
     - dupl
     - errcheck
-    - exportloopref
     - gocyclo
-    - gofmt
-    - gosimple
     - govet
     - ineffassign
     - staticcheck
-    - typecheck
```

Comment:
Have typecheck, gosimple, and exportloopref been removed from the new version?

Reply:
As per golangci-lint documentation:
- typecheck was never a linter... no idea what it was doing.
- gosimple is now a part of staticcheck.
- exportloopref was already deprecated when we added it and is removed in a new version (seems to be a part of gocyclo now).

I saw quite a few other interesting new linters, but assessing each of them would take time...